app/assets/stylesheets/friendships.css.scss

```diff
@@ -0,0 +1,16 @@
+.friendship {
+  border-bottom: 1px dotted #d3d3d3;
+}
+
+li.pure-menu-item {
+  display: inline-block;
+  margin-right: 25px;
+}
+
+.pending {
+  border-bottom: 1px dotted #008000;
+
+  a {
+    color: #008000;
```

Comment:
Color literals like `#008000` should only be used in variable declarations; they should be referred to via variable everywhere else.